builtin/mainmenu/textures.lua

```diff
@@ -145,6 +145,28 @@ function mm_texture.set_game(identifier,gamedetails)
 	
 	local path = gamedetails.path .. DIR_DELIM .."menu" ..
 									 DIR_DELIM .. identifier .. ".png"
+
+	local count = 1
+	for i = 9, 1, -1 do
```

Comment:
It is possible to have less than 9 images. The reason I'm counting down is because of https://github.com/minetest/minetest/pull/2911#discussion_r34652047